tests/integration/adapter-tests.js

```diff
@@ -207,4 +207,97 @@ export default function adapterTests() {
 
     expect(res.ok).to.be.true;
   });
+
+  describe('expired tests', () => {
+    async function testExpiration() {
+      const { persister, recordingId } = this.polly;
+      const url = '/api';
+      let har;
+
+      // request number one - records the request
+      await this.relativeFetch(url);
+      await persister.persist();
+      har = await persister.find(recordingId);
+
+      expect(har).to.be.an('object');
+      expect(har.log.entries).to.have.lengthOf(1);
+      const prevDateTime = har.log.entries[0].startedDateTime;
+
+      // wait for the first request to expire
+      await new Promise(r => setTimeout(r, 10));
+
+      // request number two - the first request is now expired
+      await this.relativeFetch(url);
+      await persister.persist();
+      har = await persister.find(recordingId);
+
+      expect(har).to.be.an('object');
+      expect(har.log.entries).to.have.lengthOf(1);
+      const nextDateTime = har.log.entries[0].startedDateTime;
+
+      // boolean returned is true if re-record occurred
+      return prevDateTime !== nextDateTime;
+    }
+
+    beforeEach(function() {
+      this.polly.configure({
+        mode: 'replay',
+        expiresIn: '1ms',
+        matchRequestsBy: {
+          order: false
+        }
+      });
+    });
+
+    afterEach(async function() {
+      await this.polly.stop();
```

Comment:
I dont think this stop is necessary here as `setupPolly` should already do that.

Reply:
removed!